src/Extensions/Shopware/PluginCreator/template/Bootstrap.tpl

```diff
@@ -42,7 +42,7 @@ class Shopware_Plugins_<?= $configuration->namespace; ?>_<?= $configuration->nam
         $tool = new \Doctrine\ORM\Tools\SchemaTool($em);
 
         $classes = array(
-            $em->getClassMetadata('Shopware\CustomModels\<?= $configuration->name; ?>\<?= $names->camelCaseModel; ?>')
+            $em->getClassMetadata('<?= $configuration->pluginConfig['namespace']; ?>\CustomModels\<?= $configuration->name; ?>\<?= $names->camelCaseModel; ?>')
```

Comment:
The "registerCustomModels" call will register models in the "Shopware\CustomModels" namespace. Even if it is technically possible to have models in another namespace (I guess), I think you change set does not reflect this by e.g. feeding the AnnotationReader with this model namespace, if I am not mistaken. So I would suggest to keep "Shopware\CustomModels", even if the rest of your namespace are e.g. "\BestIT\SomeThing\Fancy"